build_deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# This shell script is required for the app-sre quay image pipeline.
+# The App-SRE pipeline will trigger this script. We can modify the script as we like,
+# but the final result should be that we upload a new image to the quay repository.
+
+make test
+make validate
+
+IMAGE="quay.io/app-sre/configuration-anomaly-detection"
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+
+docker build -t "${IMAGE_NAME}:latest" .
+docker tag "${IMAGE_NAME}:latest" "${IMAGE_NAME}:${IMAGE_TAG}"
+
+DOCKER_CONF="${PWD}/.docker"
```

Comment:
Not a blocker, but is this safer as it doesn't change the global docker config?

Reply:
Closed, because others told me we have to use openshift/boilerplate instead